indexer/api/models/models.go

```diff
@@ -43,3 +48,38 @@ type WithdrawalResponse struct {
 	HasNextPage bool             `json:"hasNextPage"`
 	Items       []WithdrawalItem `json:"items"`
 }
+
+// FIXME make a pure function that returns a struct instead of newWithdrawalResponse
+// newWithdrawalResponse ... Converts a database.L2BridgeWithdrawalsResponse to an api.WithdrawalResponse
+func CreateWithdrawalResponse(withdrawals *database.L2BridgeWithdrawalsResponse) WithdrawalResponse {
+	items := make([]WithdrawalItem, len(withdrawals.Withdrawals))
+	for i, withdrawal := range withdrawals.Withdrawals {
+
+		msg := withdrawal.L2BridgeWithdrawal.CrossDomainMessageHash
+		if msg == nil {
+			msg = &common.Hash{}
+		}
+
+		item := WithdrawalItem{
+			Guid:                 withdrawal.L2BridgeWithdrawal.TransactionWithdrawalHash.String(),
+			L2BlockHash:          withdrawal.L2BlockHash.String(),
+			Timestamp:            withdrawal.L2BridgeWithdrawal.Tx.Timestamp,
+			From:                 withdrawal.L2BridgeWithdrawal.Tx.FromAddress.String(),
+			To:                   withdrawal.L2BridgeWithdrawal.Tx.ToAddress.String(),
+			TransactionHash:      withdrawal.L2TransactionHash.String(),
+			Amount:               withdrawal.L2BridgeWithdrawal.Tx.Amount.String(),
+			MessageHash:          msg.String(),
```

Comment:
Should we name this `CrossDomainMessageHash` to be specific?

Reply:
Yee thats a great callout. If not mistaken wouldn't the GUID be the same thing as a withdrawal `MessageHash`? (assuming its referring to the `MessagePassed` event emitted by the `L2ToL1MessagePasser` contract)